72pe.typ

```diff
@@ -168,9 +261,130 @@
 // -----------------------------------------------------------------------------
 // Ground Checklists and Information page
 // -----------------------------------------------------------------------------
-#let ground_checklists_and_info = [
+#let ground_checklists_and_info_columns = [
```

Comment:
I think we should incorporate the changes from #68 (move checking the oil levels to the nose section, and change checking the POH to ARROW docs) into this PR.